x-pack/plugins/actions/server/config.ts

```diff
@@ -37,6 +37,8 @@ export const configSchema = schema.object({
   proxyHeaders: schema.maybe(schema.recordOf(schema.string(), schema.string())),
   proxyRejectUnauthorizedCertificates: schema.boolean({ defaultValue: true }),
   rejectUnauthorized: schema.boolean({ defaultValue: true }),
+  maxResponseContentLength: schema.number({ defaultValue: 1000000 }), // 1MB
```

Comment:
__question:__ any reason we don't want to use [`schema.byteSize`](https://github.com/elastic/kibana/tree/master/packages/kbn-config-schema#schemabytesize)?

Reply:
+1 on this.
I didn't actually know `byteSize` exists until now (thanks Oleg) but it looks like the right one to use here for sure :)

Reply:
++